app/helpers/application_helper.rb

```diff
@@ -540,4 +540,22 @@ def replacement_event_notice(event_schedule)
   def quantity_left_of(resource)
     "#{resource.quantity - resource.used}/#{resource.quantity}"
   end
+
+  def concurrent_events(event)
+    return nil unless event.program.selected_event_schedules
+    event_schedule = event.program.selected_event_schedules.find_by(event: event)
```

Comment:
What if the event is not scheduled yet, so this is nil? L550 will raise an error.